specs/meroxa/resources_create_bigquery.yaml

```diff
@@ -0,0 +1,46 @@
+---
+# The name of the workflow.
+name: Add a Google BigQuery resource to the Meroxa Platform
+# The corresponding command for the workflow. Any arguments should be surrounded with two curly braces. E.g `command {{arg}}`.
+command: |-
+    meroxa resources create {{resource_name}} --type bigquery --url "bigquery://{{gcp_project_id}}/{{gcp_dataset_name}}" --client-key "$(cat {{gcp_service_account_json_file}})"
+# Any tags that the workflow should be categorized with.
+tags:
+  - meroxa
+  - data
+  - realtime
+  - bigquery
+# A description of the workflow.
+description: "Add a Google BigQuery resource to the Meroxa Platform using the following parameters:"
```

Comment:
nit: I think this would read better without the "... using the following parameters:" suffix. It's mostly implied for every workflow and will make it harder to search for workflows both in the app and in commands.dev

Reply:
@alokedesai works for me! I was trying to make it different to the workflow name, and considering it lived above the parameters thought that could be a nice addition, but I trust your guidance! Will get that udpated.